internal/services/containers/kubernetes_fleet_manager_resource_gen.go

```diff
@@ -51,6 +49,7 @@ func (r KubernetesFleetManagerResource) Arguments() map[string]*pluginsdk.Schema
 		},
 		"resource_group_name": commonschema.ResourceGroupName(),
 		"hub_profile": {
+			Deprecated: "`hub_profile` is currently not functional and is not be passed to the API",
```

Comment:
could we please link to some offical documentation explaining this/indicating this?